staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -47,6 +47,13 @@ const (
 	//
 	// Enables compression of REST responses (GET and LIST only)
 	APIResponseCompression utilfeature.Feature = "APIResponseCompression"
+
+	// owner: @k82cn
+	// alpha: v1.8
+	//
+	// Taint nodes based on their condition status for 'NetworkUnavailable',
+	// 'MemoryPressure', 'OutOfDisk' and 'DiskPressure'.
+	TaintNodesByCondition utilfeature.Feature = "TaintNodesByCondition"
```

Comment:
This is present in the generic api-server: https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go#L166.